data_storage_models/data_storage_models.md

```diff
@@ -22,33 +29,30 @@ After completion of this module, learners will be able to:
 
 @end
 
-link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles.css
+good_first_module: false
+data_task: data_management
+coding_required: false
 
-script: https://kit.fontawesome.com/83b2343bd4.js
+@sets_you_up_for
 
--->
-
-# Types of Data Storage Models
-
-<div class = "overview">
-
-## Overview
+@end
 
-@comment
+@depends_on_knowledge_available_in
 
-**Is this module right for me?** @long_description
+@end
 
-**Estimated time to completion:** @estimated_time
+@version_history 
+No previous versions.
```

Comment:
I think we must be missing content here. There should be a link to each minor update, right? So 1.0.x should be listed.